wazuh/wazuh
# Solved dup ids on SCA policies, and improve YML format.

|Related issue|
|---|
|#14346|

QA Issue:
- https://github.com/wazuh/wazuh-qa/issues/3393

This PRs applies changes to SCA policies to solve issues with duplicated check IDs between policy files. Also, we apply YML formatting to all the files. Some typos were solved, as described in the table below.

- [X] Verify there are no duplicated checks IDs after change is done.

```
ruleset/sca # find . -name '*.yml' | xargs cat| grep -e "^  \- id: " | awk {'print $3'} | uniq -c | grep -ve "   1 "
ruleset/sca # 
```

Changes done for each SCA policy file:

| ID    | SCA                                                    | Note                           |
| ----- | ------------------------------------------------------ | ------------------------------ |
| 1000  | CIS Benchmark for Debian/Linux 7                       | YML Format                     |
| 1500  | CIS Benchmark for Debian/Linux 8                       | YML Format                     |
| 2000  | CIS Benchmark for Debian/Linux 9                       | YML Format                     |
| 2500  | CIS Benchmark for Debian/Linux 10                      | YML Format                     |
| 3000  | System audit for Unix based systems                    | YML Format                     |
| 3500  | CIS Benchmark for Red Hat Enterprise Linux 5           | YML Format                     |
| 4000  | CIS Benchmark for Red Hat Enterprise Linux 6           | YML Format                     |
| 4500  | CIS Benchmark for Red Hat Enterprise Linux 7           | YML Format                     |
| 5000  | CIS Benchmark for Red Hat Enterprise Linux 8           | YML Format                     |
| 5500  | CIS Benchmark for CentOS 6                             | YML Format                     |
| 6000  | CIS Benchmark for CentOS 7                             | YML Format                     |
| 6500  | CIS Benchmark for CentOS Linux 8                       | YML Format                     |
| 7000  | CIS SUSE Linux Enterprise 11 Benchmark                 | YML Format                     |
| 7500  | CIS SUSE Linux Enterprise 12 Benchmark                 | YML Format                     |
| 8000  | CIS Benchmark for Oracle Solaris 11                    | YML Format                     |
| 8500  | CIS Apple macOS 10.11 Benchmark                        | YML Format                     |
| 9000  | CIS Apple macOS 10.12 Benchmark                        | YML Format                     |
| 9500  | CIS Apple macOS 10.13 Benchmark                        | YML Format                     |
| 10000 | CIS Apache HTTP Server 2.4 Benchmark                   | YML Format                     |
| 10500 | CIS Benchmark for Oracle MySQL Community Server 5.6    | YML Format                     |
| 11000 | CIS Benchmark for Oracle MySQL Enterprise 5.6          | YML Format                     |
| 11500 | CIS Microsoft SQL Server 2012                          | YML Format                     |
| 12000 | CIS Microsoft SQL Server 2014                          | YML Format                     |
| 12500 | CIS Microsoft SQL Server 2017                          | YML Format                     |
| 13000 | CIS Microsoft SQL Server 2016                          | YML Format                     |
| 13500 | CIS Microsoft SQL Server 2019                          | YML Format                     |
| 14000 | System audit for web-related vulnerabilities           | YML Format                     |
| 15000 | CIS Benchmark for Windows 2012 R2                      | YML Format                     |
| 15500 | CIS Benchmark for Windows 10 Enterprise (Release 1803) | IDs change                     |
| 16000 | CIS Benchmark for Windows Server 2016                  | YML Format                     |
| 16500 | CIS Benchmark for Windows Server 2019                  | YML Format and Headers updated |
| 17000 | CIS Apple macOS 10.14 Benchmark                        | YML Format                     |
| 17500 | CIS Apple macOS 10.15 Benchmark                        | YML Format                     |
| 18000 | CIS Apple macOS 11.x Benchmark                         | YML Format and Headers updated |
| 18500 | CIS benchmark for Ubuntu Linux 18.04 LTS               | YML Format                     |
| 19000 | CIS benchmark for Ubuntu Linux 20.04 LTS               | No action                      |
| 19500 | CIS benchmark for Ubuntu Linux 14.04 LTS               | YML Format and IDs change      |
| 20000 | CIS Benchmark for Amazon Linux 1                       | YML Format                     |
| 20500 | CIS Benchmark for Amazon Linux 2                       | YML Format                     |
| 21000 | CIS SUSE Linux Enterprise 15 Benchmark                 | YML Format and IDs change      |
| 21500 | CIS benchmark for Ubuntu Linux 16.04 LTS               | YML Format and IDs change      |
| 22000 | CIS Benchmark for Microsoft IIS 10                     | YML Format                     |
| 22500 | CIS MongoDB 3.6 Benchmark                              | YML Format and IDs change      |
| 23000 | CIS Benchmark for NGINX version 1.14.0                 | YML Format                     |
| 23500 | CIS Benchmark for Oracle Solaris 11.4                  | YML Format and IDs change      |
| 24000 | CIS Benchmark for PostgreSQL 13                        | YML Format                     |
| 24500 | CIS Benchmark for Oracle Database 19c                  | YML Format                     |
| 25000 | CIS HP-UX 11i Benchmark                                | No action                      |
| 26000 | CIS Microsoft Windows 11 Enterprise Benchmark          | YML Format                     |
| 27000 | CIS Microsoft Windows Server 2022                      | YML Format                     |
| 28000 | CIS Red Hat Enterprise Linux 9                         | YML Format                     |
| 28500 | CIS Ubuntu Linux 22.04                                 | No action                      |
| 29000 | CIS Apple macOS 12.0 Monterey                          | Headers updated                |
